setup-colab-env.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -ue
+
+# Check if script is run inside Google Colab.
+[ -z "$COLAB_GPU" ] && echo "Exiting: Seems like you are not running this script inside Google Colab." && exit 1;
```

Comment:
This check didn’t necessarily work for me locally on macOS. Instead I received: "bash: line 6: COLAB_GPU: unbound variable"

Reply:
I guess this must be the same before and after this rebase? Could be tracked in a separate issue/PR if you think it's worth it.